templates/form.html: Add Github fork me ribbon

Chirag0111 opened the pull request:
This PR adds a Github fork me ribbon in form.html.

Closes https://github.com/manu-chroma/username-availability-checker/issues/126

Chirag0111 committed:
templates/form.html: Add GitHub fork me ribbon

templates/form.html: Add Github fork me ribbon

This PR adds a Github fork me ribbon in form.html.

Closes https://github.com/manu-chroma/username-availability-checker/issues/126

utkarsh2102 reviewed (changes requested):
Hey,
- Changes you made aren't really working. Did you test them locally?
- If possible, let's work on the commit message and mend it as per the [guidelines](https://coala.io/commit).
- Also, indent the code properly. That really helps.
- Include the snippets of the local test in the comments before submitting it for the review again. We'd know if that works ;)
- I'm extending the deadline by 24 hours. Please submit the same at the earliest. 
- Ping me on Gitter, Zulip or Telegram, if you need any help.

manu-chroma commented:
this is duplicated code from ``status.html``

We can avoid duplicating by refactoring github icon css into main css file.

utkarsh2102 reviewed (approved):
Except for changing the branch, the changes look good to me.
I think he's having a little trouble with rebasing, that won't be a problem, I guess.
Also, the line exceeds by the usual length, but that's okay, I guess?
The changes are working fine.

li-boxuan commented on templates/form.html:
```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html>
+
```
Unnecessary change?

manu-chroma reviewed (changes requested):
github ribbon css is present in both ``status.html`` and ``form.html``

Please extract it out to ``main.css``

Chirag0111 commented:
@utkarsh2102, I have done the work.
- Added proper indentation, as asked by @utkarsh2102 and @jayvdb.
- Extracted css to `main.css`.

Please check the same.

utkarsh2102 reviewed (changes requested):
I can't seem to find the hover effect working when tested locally.
Over to @manu-chroma.
Indentation looks fine now.

manu-chroma committed:
stylelint fixes

utkarsh2102 commented:
Oh, that's awesome.
I will ask him to do so and would approve the task once he submits the same.

Chirag0111 commented:
Approved.

Chirag0111 commented:
Done.

![image](https://user-images.githubusercontent.com/40314084/48303624-514e1f80-e532-11e8-95c7-b8e2bd14155f.png)

manu-chroma commented:
:tada:
